src/ng/directive/ngModelOptions.js

```diff
@@ -0,0 +1,378 @@
+'use strict';
+
+/* global $DefaultModelOptionsProvider: true */
+
+/**
+ * @ngdoc provider
+ * @name $defaultModelOptionsProvider
+ * @description
+ *
+ * Here, you can change the default settings from which {@link ngModelOptions}
```

Comment:
Actually I don't want them to change defaults - ever. That would put us back in the same position as before where application developers can break library components that expect defaults to be a certain value.

Instead if one wants to change stuff globally then they must put an `ngModelOptions` directive on the same element as the `ng-app` attribute. Then use `"*": "$inherit"`